cohorts/load.py

```diff
@@ -1219,7 +1253,6 @@ def summarize_provenance_per_cache(self):
         frame for this cohort.
         """
         provenance_summary = {}
-        df = self.as_dataframe()
```

Comment:
I was removing this as they showed up as unused, but now seeing that we are relying on a side-effect of these functions to fill in the value?